code/modules/projectiles/gun.dm

```diff
@@ -61,13 +61,14 @@
 	var/recoil = 0		//screen shake
 	var/silenced = 0
 	var/muzzle_flash = 3
-	var/accuracy = 0   //accuracy is measured in tiles. +1 accuracy means that everything is effectively one tile closer for the purpose of miss chance, -1 means the opposite. launchers are not supported, at the moment.
+	var/accuracy = 0   //Accuracy is measured in percents. +15 accuracy means that everything is effectively one tile closer for the purpose of miss chance, -15 means the opposite. launchers are not supported, at the moment.

```

Comment:
This change requires all our guns to be updated. All accuracy multiplied by 15.

Reply:
I updated our guns.